packages/interledger/src/admin-api/schema.graphql

```diff
@@ -1,22 +1,437 @@
 type Query {
-  accounts: String
+  "Fetch a page of Interledger accounts."
+  ilpAccounts(
+    "Paginating forwards: the cursor before the the requested page."
+    after: String
+    "Paginating backwards: the cursor after the the requested page."
+    before: String
+    "Paginating forwards: The first **n** elements from the page."
+    first: Int
+    "Paginating backwards: The last **n** elements from the page."
+    last: Int
+  ): IlpAccountsConnection!
+
+  "Get an Interledger account by ID."
+  ilpAccount(id: ID!): IlpAccount!
+
+  "Get a trustline by ID."
+  trustline("The ID of the trustline to get." id: ID!): Trustline!
+
+  "Get a webhook by ID."
+  webhook("The ID of the webhook to get." id: ID!): Webhook!
+
+  "Get a deposit by ID."
+  deposit("The ID of the deposit to get." id: ID!): Deposit!
+
+  "Get a withdrawal by ID."
+  withdrawal("The ID of the withdrawal to get." id: ID!): Withdrawal!
+}
+
+type Mutation {
+  "Create Interledger account"
+  createIlpAccount: CreateIlpAccountMutationResponse!
+
+  "Update Interledger account"
+  updateIlpAccount: UpdateIlpAccountMutationResponse!
+
+  "Delete Interledger account"
+  deleteIlpAccount: DeleteIlpAccountMutationResponse!
+
+  "Create Interledger sub-account"
+  createIlpSubAccount(superAccountId: ID!): CreateIlpSubAccountMutationResponse!
+
+  "Transfer between Interledger accounts"
+  transfer(
+    originAmount: Int!
```

Comment:
graphQL ints are typically limited to 32 bits: https://graphql.org/learn/schema/#scalar-types

We should use string numbers instead, or define a custom scalar type that serializes to bigints.

Reply:
I think it's probably best to create a custom scalar. I'll create a `UInt64` that maps to a `BigInt`.